### ISSUE TITLE
Suggestions - if still being worked on

### ISSUE THREAD

**Issue by Anthonyg5005** [TheLastMillennial/HD-Picture-Viewer#21]
I'm using the version in the bin folder and I want to suggest being able to switch the amount of pixels to scroll and zoom in. for example scrolling 1 pixel at a time. I'd also like to see a button for fitting the image on screen if possible

**Comment by TheLastMillennial** [comment]
Zooming in a specific amount is a far out feature.

Zoom to fit is already in the latest pre-release. I think it's [Window].

**Comment by Anthonyg5005** [comment]
Yeah I noticed it was. I think it is if the image is at least the same size as the screen. I think I was making them smaller and it didn't work. Also seems limited to 3 images but it's still just a pre-release

**Comment by asdasdasd1214** [comment]
suggestions - when you are on file #1 it should bring you to the bottom of the list if you click the up arrow (since theres nothing above it currently nothing happens), and also folder sorting for images

**Comment by TheLastMillennial** [comment]
A setting to zoom in by a specific amount isn't planned for now. Zoom now scales by 10% in the latest build

Zoom to fit has been implemented.

Scroll jumping has been implemented.

Far more than 3 pictures are supported.

Closing
